ios/RNSSearchBar.mm

```diff
@@ -143,6 +144,11 @@ - (void)setHideWhenScrolling:(BOOL)hideWhenScrolling
   _hideWhenScrolling = hideWhenScrolling;
 }
 
+- (void)setRemoveTitle:(BOOL)removeTitle
+{
+  _removeTitle = removeTitle;
+}
```

Comment:
As long as there is no further logic this setter function is redundant, as it will be auto-created by compiler. I believe we can safely delete it.